src/app/core/services/personal-cards.service.ts

```
@@ -350,7 +350,22 @@
     return this.getBankTransactions(params, usePlatformApi).pipe(map((res) => res.count));
   }
 
-  fetchTransactions(accountId: string): Observable<ApiV2Response<PersonalCardTxn>> {
+  fetchTransactionsPlatform(accountId: string): Observable<PlatformApiResponse<{}>> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)